### ISSUE TITLE
redis_hacks.rb broken with redis 2.2.0 gem

### ISSUE THREAD

**Issue by agibralter** [thoughtbot/daikon#2]
```
app/releases/a8eb262074e9f1436f4aaaf06931cd4d5676735c/vendor/bundle/ruby/1.8/gems/daikon-0.8.7/lib/daikon/redis_hacks.rb:36: Connection is not a class (TypeError)
app/releases/a8eb262074e9f1436f4aaaf06931cd4d5676735c/vendor/bundle/ruby/1.8/gems/daikon-0.8.7/bin/../lib/daikon.rb:19:in `require'
app/releases/a8eb262074e9f1436f4aaaf06931cd4d5676735c/vendor/bundle/ruby/1.8/gems/daikon-0.8.7/bin/../lib/daikon.rb:19
app/releases/a8eb262074e9f1436f4aaaf06931cd4d5676735c/vendor/bundle/ruby/1.8/gems/daikon-0.8.7/bin/daikon:3:in `require'
app/releases/a8eb262074e9f1436f4aaaf06931cd4d5676735c/vendor/bundle/ruby/1.8/gems/daikon-0.8.7/bin/daikon:3
app/releases/a8eb262074e9f1436f4aaaf06931cd4d5676735c/vendor/bundle/ruby/1.8/bin/daikon:19:in `load'
app/releases/a8eb262074e9f1436f4aaaf06931cd4d5676735c/vendor/bundle/ruby/1.8/bin/daikon:19
```

**Comment by agibralter** [comment]
Hmm can you release different versions of the gem to support different versions of Redis?

**Comment by shedd** [comment]
I guess that's why I can't install the gem - Bunder gives me the following error:

Bundler could not find compatible versions for gem "redis":
  In Gemfile:
    daikon depends on
      redis (~> 2.1.1)

```
redis (2.2.1)
```

Here's another vote for supporting the 2.2.x version of the Redis gem.

**Comment by qrush** [comment]
Hey folks,

Sorry I didn't see this. Daikon now supports Redis 2.2, just upgrade to the latest version (as of this post, ~> 0.8.13).

Backstory on this: The redis-rb gem > 2.0 hasn't supported Redis 1.2.6's MONITOR protocol, so I had to hack that in. Of course, they changed APIs in redis-rb 2.2+ (no thanks to semver) so I had to re-hack it to make it work.

If the latest gem doesn't work crack open a new issue or bug me at support@radishapp.com.
